llama_index/indices/multi_modal/retrievers.py

```diff
@@ -0,0 +1,172 @@
+"""Base vector store index query."""
+
+
+from typing import Any, Dict, List, Optional
+
+from llama_index.constants import DEFAULT_SIMILARITY_TOP_K
+from llama_index.data_structs.data_structs import IndexDict
+from llama_index.indices.base_retriever import BaseRetriever
+from llama_index.indices.query.schema import QueryBundle
+from llama_index.indices.utils import log_vector_store_query_result
+from llama_index.indices.vector_store.base import VectorStoreIndex
+from llama_index.schema import NodeWithScore, ObjectType
+from llama_index.vector_stores.types import (
+    MetadataFilters,
+    VectorStoreQuery,
+    VectorStoreQueryMode,
+    VectorStoreQueryResult,
+)
+
+
+class MultiModalVectorIndexRetriever(BaseRetriever):
```

Comment:
to confirm, this retriever is only for text index and as a placeholder right
We can first merge this one or remove it first?

Reply:
It's just a placeholder for now, I can remove it :)